### ISSUE TITLE
Turning off all buttons does not remove the toolbar

### ISSUE THREAD

**Issue by ishanuda** [stephanrauh/ngx-extended-pdf-viewer#198]
Hi,

I tried to turn off the toolbar by disabling all the buttons setting to false, still shows the empty toolbar.
When I re-size the screen or open/resize the development tools space below in the browser,
The empty toolbar disappears.

That means any changes to the screen size make the toolbar disappear.

Tested on Angular 8 with Angular Material custom theme applied
"ngx-extended-pdf-viewer": "^2.0.0-alpha.2",

**Feature Request:**
**Also, I would like to suggest adding an attribute `[showToolbar]="true/false"` as an easy way to turn on/off the toolbar without turning off all the buttons available.**

> NOTE: I did not see any attribute like above in the documentation or in the API.

Regards

**Comment by stephanrauh** [comment]
The feature works, as you can see in https://pdfviewer.net/hiding-buttons. However, I've found some combinations that seem to be broken. I'll have a look at this. 
You may also want to deactivate `showSecondaryToolbarButton`. 

As for the feature request: you're right. Consider it done!

**Comment by ishanuda** [comment]
Hi,
Thank you for the quick reply,

My settings are as follows:

```
<ngx-extended-pdf-viewer
    [src]="url"

    [showBookmarkButton]="false"
    [showDownloadButton]="false"
    [showFindButton]="false"
    [showHandToolButton]="false"
    [showOpenFileButton]="false"
    [showPagingButtons]="false"
    [showPresentationModeButton]="false"
    [showPrintButton]="false"
    [showPropertiesButton]="false"
    [showRotateButton]="false"
    [showScrollingButton]="false"
    [showSecondaryToolbarButton]="false"
    [showSidebarButton]="false"

    [showSidebarOnLoad]="false"

    [showSpreadButton]="false"
    [showZoomButtons]="false"

    [filenameForDownload]="false"

    [showBorders]="showBorders"
    useBrowserLocale="true"
    delayFirstView="1000"
    textLayer="false"

    [height]="height">
```

But still, the result is the same.

regards.

**Comment by stephanrauh** [comment]
Related, but not identical to your issue: #200

**Comment by stephanrauh** [comment]
Making the toolbars customizable didn't make this issue any simpler. Now we don't know which items are in the toolbar. So the only option is to check it dynamically via JavaScript. Let's shift this issue to version 2.1+.

**Comment by stephanrauh** [comment]
An all-new implementation has landed with version 2.1.x. Now the toolbars are shown and hidden in a predictable and reliable way.
